README.md

```diff
@@ -79,11 +79,13 @@ container to the docker group for socket permissions.
 
 ### `token`
 
-[Generate a Personal Access Token](https://github.com/settings/tokens), with the `repo:public_repo` scope for only public repositories or the `repo` scope for public and private repositories, and add it to _Secrets_ (repository settings) as `RENOVATE_TOKEN`.
+[Generate a Personal Access Token (classic)](https://github.com/settings/tokens), with the `repo:public_repo` scope for only public repositories or the `repo` scope for public and private repositories, and add it to _Secrets_ (repository settings) as `RENOVATE_TOKEN`.
 You can also create a token without a specific scope, which gives read-only access to public repositories, for testing.
 This token is only used by Renovate, see the [token configuration](https://docs.renovatebot.com/self-hosted-configuration/#token), and gives it access to the repositories.
 The name of the secret can be anything as long as it matches the argument given to the `token` option.
 
+Note that Renovate does *not* currently support using [Fine-grained Personal Access Tokens](https://github.com/settings/tokens?type=beta).
```

Comment:
It's not a matter of Renovate not accepting them, it's that they're not capable enough.

Reply:
Updated to state reason for non-support